src/views/shared/network-box/network-box.styles.ts

```diff
@@ -29,10 +29,16 @@ export const useNetworkBoxStyles = createUseStyles((theme: Theme) => ({
     alignItems: "center",
```

Comment:
The buttons of the box show wrap:

<img width="263" alt="image" src="https://user-images.githubusercontent.com/15896177/229824807-83abf923-2c04-483b-b345-cd2f912e4d7b.png">